docusaurus/docs/api-endpoints/api/user/get-settings.md

```diff
@@ -0,0 +1,7 @@
+# GET /api/user/settings
+
+Get user settings from the database.
```

Comment:
Is there any format for user settings?

Reply:
Added a sentence saying that the backend does not care about the stucture of the JSON.